chameleon/settings.py

```diff
@@ -719,9 +719,17 @@
 CSP_FRAME_ANCESTORS = "'self'"  # Similar to X-Frame-Options: SAMEORIGIN
 CSP_SCRIPT_SRC = ["'self'", 'https://www.google-analytics.com/', "'unsafe-inline'"]
 CSP_FONT_SRC = ["'self'", 'https://fonts.gstatic.com/']
+CSP_IMG_SRC = ["'self'", '*.googleusercontent.com', "'unsafe-inline'"]
```

Comment:
I'm not sure that we need `unsafe-inline` here, I'm not sure how that even applies to img elements in CSP actually 🤔